examples/example-gateway/config/production.yaml

```diff
@@ -122,4 +122,7 @@ circuitbreaking-configurations:
       sleepWindowInMilliseconds: 1000
       errorPercentThreshold: 30
       requestVolumeThreshold: 25
-      maxConcurrentRequests: 30
\ No newline at end of file
+      maxConcurrentRequests: 30
+
+service.shadow.env.override.enable: true
+dedicated.tchannel.client: true
```

Comment:
qq: How have we handled the default behaviour? Does it remain the same? Should the example be for default / old behaviour or new behaviour?

Reply:
- Default behaviour: Same as before
- `Example-gateway` example has the new behaviour. `Selective-gateway` example has the default behaviour